uui/components/layout/VirtualList.tsx

```diff
@@ -118,7 +118,15 @@ const VirtualListView = React.forwardRef<HTMLDivElement, VirtualListViewProps>((
             >
                 { props.children }
             </div>
-            <Blocker isEnabled={ props.isLoading } />
+            <Blocker
+                isEnabled={ props.isLoading }
+                rawProps={ { 
+                    role: 'status',
+                    'aria-live': 'polite',
+                    'aria-busy': props.isLoading ? 'true' : 'false',
+                    'aria-label': props.isLoading ? 'Loading' : '',
+                } }
```

Comment:
Can we put these aria attributes directly into the Blocker component? We can use 'isEnabled' there, the same way we use isLoading here.